temoa/extensions/myopic/myopic_sequencer.py

```
@@ -541,24 +568,26 @@ def clear_results_after(self, period):
         logger.debug('Clearing periods %s+ from output tables', period)
 
         for table in self.tables_with_period:
+            scenario_name = self.config.scenario if self.config else None
             try:
                 self.cursor.execute(
                     f'DELETE FROM {table} WHERE period >= (?) and scenario = (?)',
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)